1-exercises/A-reading-list/index.html

```diff
@@ -11,6 +11,7 @@
         DO NOT EDIT.
         Your work should be put in here using JavaScript.
       -->
+      <script src="script.js"></script>
```

Comment:
you don't need to add this, it's already added for you at the bottom of the html file